src/templates/Section.tsx

```diff
@@ -1,17 +1,111 @@
 import React from 'react'
+import { IArticle, IMostReadArticle } from '../types'
+import Layout from '../components/layout'
+import SEO from '../components/seo'
+import Footer from '../components/Footer'
+import { Row, Col, Container, Image } from 'react-bootstrap'
+import s from 'styled-components'
+import RightCol from '../components/home/RightCol'
+import { generateSlug, IMAGE_URL } from '../utils/helperFunctions'
+import { StyledLink } from '../components/shared'
 
-const Section = () => {
-  // {mostReadUTB.map(article => (
-  //   <Row style={{ borderBottom: '1px solid #EBEBEB', padding: '1em 0', marginRight: '1em' }}>
-  //     <Col>
-  //       <img src={article.image} />
-  //     </Col>
-  //     <Col>
-  //     <p> <strong dangerouslySetInnerHTML={{ __html: article.ogTitle }}/> </p>
-  //     <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
-  //     </Col>
-  //   </Row>
-  // ))}
+
+interface ISectionProps {
+  pageContext: {
+    filteredArticles: IArticle[]
+    mostReadDP: IMostReadArticle[]
+    section: string
+    topArticles: IArticle[]
+    centerpiece: IArticle
+  }
+}
+
+const SubHeader = s.h3`
+  color: #aa1e22;
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 1.0;
+  margin: 0.5em 0;
+  color: ${({ color }) => color};
+`
+
+const Filler = s.div`
+  margin-top: 1em;
+  margin-right: 1em;
+  margin-bottom: 1em;
+  height: 90px;
+  background-color: #DFF3DB;
+`
+
```

Comment:
the `ISectionProps` interface should be moved to here, below the styled components